ldes-server-domain/src/main/resources/viewShaclShape.ttl

```diff
@@ -0,0 +1,77 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix tree: <https://w3id.org/tree#> .
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
+
+tree:ViewDescription
+	sh:order 1;
+  	a sh:NodeShape ;
+	sh:property [
+      	sh:path tree:fragmentationStrategy ;
+		sh:name "List of fragmentations";
+        sh:description "Ordered list of fragmentations how a member for this view should be processed";
+    	sh:or (tree:GeospatialFragmentation tree:PaginationFragmentation tree:SubstringFragmentation tree:TimebasedFragmentation) ;
+        sh:minCount 1;
+    ].
+
+tree:GeospatialFragmentation
+	sh:order 2;
+    a sh:NodeShape;
+	sh:class tree:GeospatialFragmentation ;
+	sh:targetClass tree:GeospatialFragmentation ;
+    sh:property [
+      	sh:name "Max Zoom";
+      	sh:description "Maximal zoom level the fragmentation should go";
+        sh:path tree:maxZoom ;
+        sh:datatype xsd:integer ;
+        sh:minInclusive 0 ;
+        sh:maxInclusive 21;
```

Comment:
are you sure about this 21? A random search resulted in 23 for me https://developers.arcgis.com/documentation/mapping-apis-and-services/reference/zoom-levels-and-scale/